src/interoperability.md

```diff
@@ -1 +1,19 @@
-Contents will be added as part of https://github.com/sora-xor/sora-docs/pull/44
+# Interoperability
+
+This article is about the HASHI bridge - a decentralized two-way bridge connecting SORA and other blockchains (such as Ethereum and Polkadot). HASHI means bridge in Japanese, and it is the plumbing that links many different chains, like pipes connecting the world's assets. The initial version of this bridge was a significant component of the plans for the beta launch of the SORA network in Q1 2021.
+
+## History
+
+The SORA blockchain serves as an ecosystem for robust DeFi projects like SORA's decentralized global economic system and [Polkaswap](./polkaswap), a DEX specifically designed for an interoperable future. In line with this vision, we recognize that blockchain maximalism is no longer viable, and we acknowledge the importance of secure, fast, and efficient asset migration between blockchains.
+
+As an initial step towards achieving this, we have developed the source code for Ethereum Bridge. In addition to Ethereum, SORA aims to establish connections with various ecosystems. Being built on Substrate, it is advantageous to utilize the built-in cross-chain transfer capabilities to facilitate communication with projects running on the Polkadot and Kusama ecosystems. To achieve this, the [Substrate bridge](./substrate-bridge) was developed.
+
+With multiple bridges already in operation, it becomes necessary to streamline the bridge interface to ensure consistent software components for cross-chain transfers across different heterogeneous blockchains. As a solution, the HASHI bridge is aggregating several bridge technologies to serve as the gateway between the SORA mainnet and the external world.
+
+The HASHI bridge currently comprises two bridges: the Federated EVM Bridge, and the Federated Substrate Bridge. As currently [Polkadot's BEEFY](https://wiki.polkadot.network/docs/learn-consensus#bridging-beefy) protocol is still in development, HASHI uses a federated approach for now. Trusted bridges rely on a central entity or a group of trusted parties to help move assets and data between different blockchains. On the other hand, trustless bridges aim to do away with the need for intermediaries by using smart contracts or decentralized protocols to enable cross-chain transfers. Federated bridges combine aspects of both trusted and trustless bridges. Instead of relying on a single trusted entity, they involve a group of trusted relayers working together to manage and secure cross-chain transactions.
```

Comment:
```suggestion
The HASHI bridge currently comprises two bridges: the Federated EVM Bridge, and the Federated Substrate Bridge. HASHI uses a federated architecture for now before becoming fully trustless, as currently [Polkadot's BEEFY](https://wiki.polkadot.network/docs/learn-consensus#bridging-beefy) protocol is still in development.

Federated bridges combine aspects of both trusted and trustless bridges. Instead of relying on a single trusted entity, they involve a group of trusted relayers working together to manage and secure cross-chain transactions.

Trusted bridges rely on a central entity or a group of trusted parties to help move assets and data between different blockchains. 

On the other hand, trustless bridges aim to do away with the need for intermediaries by using smart contracts or decentralized protocols to enable cross-chain transfers. ```

Reply:
What is the reason for dividing the paragraph into several smaller ones, where some of them are containing only a single sentence? Since this content falls under the _History_ section, I believe it should be kept together as a group. Furthermore, in academic writing guidelines, the use of short paragraphs is not typically recommended and I think this guideline should also apply to this documentation, as it is not a promotional material.

Reply:
@bgodlin this is still unresolved